### ISSUE TITLE
[DataScience_with_Python]: Create CheatSheet for Matplotlib library of Python

### ISSUE THREAD

**Issue by singhmansi25** [girlscript/winter-of-contributing#6908]
### Description

**Description :** This issue deals with the documentation for cheat-sheet for matplotlib library of Python.
The readme.md will contain the codes and a .ipynb file will contains all the demonstration of all the codes.

### Domain

Datascience with Python

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

**Comment by Ruchika-20** [comment]
I would like to work on this issue

**Comment by singhmansi25** [comment]
@Ruchika-20 
I have opened the issue, so would like to work on it.
You can add changes later, if you feel so.
Thanks.

**Comment by Ruchika-20** [comment]
We can open the issues and work on them?

Then I would too like todo the same as I'm unable to get issues

**Comment by singhmansi25** [comment]
Yes @Ruchika-20 , you can create issues on your own. and work on them.

**Comment by Ruchika-20** [comment]
Do I need to get the permission of supervisors or inform them....whya do I
have to do?

**Comment by singhmansi25** [comment]
Yes you need to create issue
and ask supervisor to assign issue to you
and then work on it

**Comment by Ruchika-20** [comment]
Okaaayyy and that issue can be any topic of DSWP?

**Comment by singhmansi25** [comment]
you can check the repo and create accordingly

**Comment by prathimacode-hub** [comment]
@singhmansi25 @Ruchika-20, process is wrong. You can create issue only if its approved that was when I was into limited access due to laptop issue. But it resumed back to normal. Since @singhmansi25 has raised issue then it's considered. But now it's back to normal, issues would be created by mentors and supervisor, you have to express interest and get it assigned. If you would like to work on any issue, you need to provide details in channel. If it's approved, you will be reverted back with a link to claim your issue within 6-8 hours of assigning. This is the process.

**Comment by prathimacode-hub** [comment]
Issue assigned to @singhmansi25 for documentation

**Comment by Ruchika-20** [comment]
Okkk

**Comment by singhmansi25** [comment]
So, from now on we can't create new issues ?
@prathimacode-hub

**Comment by singhmansi25** [comment]
@prathimacode-hub, will you close this issue or do I need to close it.

**Comment by prathimacode-hub** [comment]
Don't close. @singhmansi25

**Comment by ashish-reddy-20-08** [comment]
@prathimacode-hub , Mam
I am M. Ashish Reddy. I would like to do video on this issue.
Batch number: 5
serial number:174.
Approach: A detailed explanation with examples and real life uses

**Comment by prathimacode-hub** [comment]
Issue assigned to @ashish-reddy-20-08 for video contribution
